app/services/catalog/create_approval_request.rb

```diff
@@ -0,0 +1,66 @@
+require 'approval_api_client'
+
+module Catalog
+  class CreateApprovalRequest
+    attr_reader :order
+
+    def initialize(id)
+      @order = Order.find_by!(:id => id)
+
+      ApprovalApiClient.configure do |config|
+        config.api_key['x-rh-auth-identity'] = ManageIQ::API::Common::Request.current.request_id
```

Comment:
@lindgrenj6 
This line should be replaced with 
https://github.com/ManageIQ/catalog-api/blob/c496425db1ec5b7ac2abdd342526d4627fdcd860/lib/topological_inventory.rb#L29
Also you need to set the credentials for the dev environment by calling
https://github.com/ManageIQ/catalog-api/blob/c496425db1ec5b7ac2abdd342526d4627fdcd860/lib/topological_inventory.rb#L24

Reply:
Updated to include this code.